source/frontend/src/components/common/buttons/RefreshButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { TbRefresh } from 'react-icons/tb';
+
+import TooltipWrapper from '../TooltipWrapper';
+import { Button, ButtonProps } from './Button';
+
+/**
+ * Button displaying a refresh/recycle icon, used to reset form data.
+ * @param param0
+ */
+const RefreshButton: React.FC<React.PropsWithChildren<ButtonProps>> = ({ ...props }) => {
+  return (
+    <TooltipWrapper tooltipId="btn-refresh-tooltip" tooltip="Refresh">
+      <Button
+        data-testid="refresh-button"
```

Comment:
given that this button will be placed inside other screens, please make this id/testid/title more specific

Reply:
updated